Himanshu_Sharma/form_using_bootstrap/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assets/styles/main.css">
+</head>
+<body>
+    <div style="height: 100vh;" class="d-flex flex-column justify-content-center bg-dark">
```

Comment:
get rid of all style attributes in html files

Reply:
This was just an assignment for bootstrap it is not for review, I will take care of it next time, but won't bootstrap implemented like it!

Reply:
suggestion: either use `class` or `style` not both at the same time
you are violating KISS principle + breaking natural cascade from external css